kevm-pyk/src/kevm_pyk/kproj/evm-semantics/gas.md

```diff
@@ -102,23 +102,23 @@ module GAS-FEES
     imports GAS-SYNTAX
     imports SCHEDULE
 
-    syntax Gas ::= Cgascap        ( Schedule , Gas , Gas , Int )         [klabel(Cgascap),         function, total, smtlib(gas_Cgascap_Gas)  ]
-    syntax Int ::= Cgascap        ( Schedule , Int , Int , Int )         [klabel(Cgascap),         function, total, smtlib(gas_Cgascap_Int)  ]
-                 | Csstore        ( Schedule , Int , Int , Int )         [klabel(Csstore),         function, total, smtlib(gas_Csstore)      ]
-                 | Rsstore        ( Schedule , Int , Int , Int )         [klabel(Rsstore),         function, total, smtlib(gas_Rsstore)      ]
-                 | Cextra         ( Schedule , Bool , Int , Bool )       [klabel(Cextra),          function, total, smtlib(gas_Cextra)       ]
-                 | Cnew           ( Schedule , Bool , Int )              [klabel(Cnew),            function, total, smtlib(gas_Cnew)         ]
-                 | Cxfer          ( Schedule , Int )                     [klabel(Cxfer),           function, total, smtlib(gas_Cxfer)        ]
-                 | Cmem           ( Schedule , Int )                     [klabel(Cmem),            function, total, smtlib(gas_Cmem), memo   ]
-                 | Caddraccess    ( Schedule , Bool )                    [klabel(Caddraccess),     function, total, smtlib(gas_Caddraccess)  ]
+    syntax Gas ::= Cgascap        ( Schedule , Gas , Gas , Int ) [symbol(Cgascap_Gas), overload(Cgascap), function, total, smtlib(gas_Cgascap_Gas)]
```

Comment:
This could be realigned with the rest of the entries for consistency

Reply:
Like this?

![aligned](https://github.com/runtimeverification/evm-semantics/assets/10085903/cf7f0c89-e272-4b72-9e8d-b5c65b573155)

Reply:
I originally wanted to align the attributes (by moving `overload` to the end), but on my monitor it already overflows.

Reply:
Right now it looks like this:

![non-aligned](https://github.com/runtimeverification/evm-semantics/assets/10085903/750366b5-dba3-4b07-8233-7979e23b8a75)

Reply:
I think both comments are most readily resolved with a newline between 106 and 107 (to separate the overloads from the non-overloads, as you say below)

Reply:
Done.